### PR TITLE
AT_01.05.011_Header > Redirect to User Configure Page

### PR DESCRIPTION
**Pull request by DimaCurkanu**
https://trello.com/c/oIAyVUnq/2168-rf-pom010511-headerredirect-to-user-configure-page

### CODE REVIEW

**Comment by DimaCurkanu** [comment]
[[RF] POM_01.05.11 | Header>Redirect to User Configure Page](https://trello.com/c/oIAyVUnq/2168-rf-pom010511-headerredirect-to-user-configure-page)

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 1]
```diff
@@ -327,4 +328,11 @@ describe('headerAndFooter', () => {
             .should('contain', Cypress.env('local.admin.username'));
     });
 
+    it.only('AT_01.05.011 | Header>Redirect to User Configure Page', () => {
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:331]: Please delete `only`

**Review by larysareznikova — APPROVED** [review]

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 2]
```diff
@@ -16,14 +16,15 @@ import restApiDocPageData from "..//..//fixtures/pom_fixtures/restApiDocPage.jso
 import SystemLogPage from "../../pageObjects/SystemLogPage";
 import ResultSearchBoxPage from "../../pageObjects/ResultSearchBoxPage";
 import myViewData from "../../fixtures/pom_fixtures/myView.json";
+import UserConfigurePageData from "../../fixtures/pom_fixtures/userConfigurePage.json";
```
Comment (Rom-T) [inline cypress/e2e/tests/headerAndFooter.cy.js:19]: Delete please. Duplicate with line 10

File: cypress/e2e/tests/headerAndFooter.cy.js (hunk 1 above)
Comment (IrynaKolh) [inline cypress/e2e/tests/headerAndFooter.cy.js:331]: 1) delete only
2) you AT description must be connected with AC (A Breadcrumb should be including "Configure" in the trail after clicking on Dropdown Profile Configure option) -> 'AT_01.05.011 | Header>Verify that Breadcrumb includes "Configure" in the trail after redirecting to User Configure Page

Comment (IrynaKolh) [reply cypress/e2e/tests/headerAndFooter.cy.js:331]: if you wrote test for covering this AC (User should be redirect to the “Configure“ Page with endpoint /user/…/configure after clicking on Dropdown Profile Configure option) you need check URL

Comment (Rom-T) [reply cypress/e2e/tests/headerAndFooter.cy.js:331]: It all depends on how TC was written.  If the url is not checked there, then the check is not necessary.

Comment (IrynaKolh) [reply cypress/e2e/tests/headerAndFooter.cy.js:331]: This US has 3 AC, so i think need clarify what was checked.

Comment (IrynaKolh) [reply cypress/e2e/tests/headerAndFooter.cy.js:331]: TC_01.05_011Verify that the page URL is redirected to "/user/.../configure" not breadcrumbs

Comment (DimaCurkanu) [reply cypress/e2e/tests/headerAndFooter.cy.js]: o-o-o! Done!

File: cypress/e2e/tests/headerAndFooter.cy.js (hunk 2 above)
Comment (DimaCurkanu) [reply cypress/e2e/tests/headerAndFooter.cy.js:19]: Done

File: cypress/e2e/tests/headerAndFooter.cy.js (hunk 1 above)
Comment (DimaCurkanu) [reply cypress/e2e/tests/headerAndFooter.cy.js]: 1) Acceptance criteria are: User should be redirected to the “Configure“ Page with endpoint /user/…/configure after clicking on Dropdown Profile Configure option. The word 'Configure' in Breadcrumbs is used as assert only. 
2) there should be one assertion per test. Am I right or am I wrong? In this test I check that the user is redirected to the configuration page using the word "Configure"

Comment (IrynaKolh) [reply cypress/e2e/tests/headerAndFooter.cy.js]: В матрице это ТС отвечает за проверку url, а ты проверяешь брэдкрамс

Comment (DimaCurkanu) [reply cypress/e2e/tests/headerAndFooter.cy.js]: I replaced the "contain" to "have.text" for a stricter test

**Review by Rom-T — APPROVED** [review]

**Pull request merged**